CASMTRIAGE-3232: Uploading BMC Recovery Firmware into TFTP Servers (#1462)
Cray-HPE/docs-csm: operations/firmware/Upload_Olympus_BMC_Recovery_Firmware_into_TFTP_Server.md

`cray-upload-recovery-images` is a utility for uploading the BMC recovery files for ChassisBMCs, NodeBMCs, and RouterBMCs to be served by the *cray-tftp* service. The tool uses the cray cli (*fas*, *artifacts*) and *cray-tftp* to download the s3 recovery images (as remembered by FAS) then upload them into the PVC that is used by *cray-tftp*.
`cray-upload-recovery-images` should be run on every system.

> **Prerequisite:**

Reviewer: ```suggestion
## Prerequisites
```

Author: Updated

> 1. CSM software has been installed

Reviewer: Assuming the order doesn't matter here, these steps can be changed from an ordered list to an unordered list.

Author: Updated

> 1. CSM software has been installed

Reviewer: ```suggestion
1. Cray System Management (CSM) software is installed
```

Author: Updated

> + Cray System Management (CSM) software is installed

Reviewer 2: ```suggestion
* Cray System Management (CSM) software is installed.
```

Author: Updated

> 2. Cray CLI software has been installed

Reviewer: ```suggestion
2. The Cray command line interface \(CLI\) tool is initialized and configured on the system
```
This is the verbiage used in other procedures.

Author: Updated

> + The Cray Command Line Interface (CLI) tool is initialized and configured on the system

Reviewer 2: ```suggestion
* The Cray Command Line Interface (CLI) tool is initialized and configured on the system.
```

> 3. Firmware has been loaded into FAS as part of the HPC Firmware Pack (HFP) install

Reviewer: ```suggestion
3. Firmware is loaded into FAS as part of the HPC Firmware Pack (HFP) install; refer to the *HPE Cray EX System HPC Firmware Pack Installation Guide* on the HPE Customer Support Center for more information
```

Author: Updated

> + Firmware is loaded into FAS as part of the HPC Firmware Pack (HFP) install; refer to the *HPE Cray EX System HPC Firmware Pack Install Guide* on the HPE Customer Support Center for more information.

Reviewer 2: ```suggestion
* Firmware is loaded into FAS as part of the HPC Firmware Pack (HFP) install; refer to the *HPE Cray EX System HPC Firmware Pack Install Guide* on the HPE Customer Support Center for more information.
```

Author: Updated

> ### Procedure

Reviewer 2: ```suggestion
## Procedure
```

Author: Updated
